src/main/java/src/chapter7/question3/Question3.java

```diff
@@ -0,0 +1,25 @@
+package src.chapter7.question3;
```

Comment:
This is an old version of NumberDivisibility class.

Reply:
Class has been removed.